mu4/playback/iplaybackcontroller.h

```diff
@@ -31,6 +31,11 @@ class IPlaybackController : MODULE_EXPORT_INTERFACE
 {
     INTERFACE_ID(IPlaybackController)
 
+    enum CursorType {
```

Comment:
Is this enum used anywhere?

Reply:
This enum comes from git merge on rebase. It's not a part of audioengine.

Reply:
>It's not a part of the audioengine

You need to remove it for full compliance with the master

Reply:
@RomanPudashkin 
Sorry. My mistake, when I aswered first time.

It used here: https://github.com/musescore/MuseScore/blob/f647e3f34d1374a72623817c992f54bd47bab4dc/mu4/playback/internal/playbackcontroller.cpp#L46